src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Extensions/RazorCodeDocumentExtensions.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common.Extensions
 {
-    internal static class RazorCodeDocumentExtensions
+    public static class RazorCodeDocumentExtensions
```

Comment:
Would it be possible to keep this as `internal` and IVT somewhere in our code base?

Reply:
Definitely. Added the file as a shared file reference to the test project instead, hopefully that's a better alternative